src/www/js/script.js

```diff
@@ -467,86 +503,100 @@ uninstallbtn.addEventListener('click', async () => {
         installbtn.disabled = false;
         playbtn.disabled = true;
         serverbtn.disabled = true;
+        logbtn.disabled = false;
     });
 });
 
-function install () {
+function install() {
     busy = true;
     playbtn.disabled = true;
     serverbtn.disabled = true;
     if (fs.existsSync(path.join(__dirname, '..', '..', 'Server') || path.join(__dirname, '..', '..', 'Client'))) {
-        Notification.show('information', 'Removing old files...');
-        fs.rm(path.join(__dirname, '..', '..', 'Server'), { recursive: true, force: true }, (err) => {
+        Notification.show('information', 'Removing old files');
+        fs.rm(path.join(__dirname, '..', '..', 'Server'), {
+            recursive: true,
+            force: true
+        }, (err) => {
             if (err) {
                 Notification.show('error', 'Failed to remove old files');
             }
         });
-        fs.rm(path.join(__dirname, '..', '..', 'Client'), { recursive: true, force: true }, (err) => {
+        fs.rm(path.join(__dirname, '..', '..', 'Client'), {
+            recursive: true,
+            force: true
+        }, (err) => {
             if (err) {
                 Notification.show('error', 'Failed to remove old files');
             }
         });
     }
     disableAll();
     // Download server files
-    Notification.show('information', 'Downloading Server files...');
+    Notification.show('information', 'Downloading Server files');
     download({
         url: 'https://osfr.editz.dev/Server.zip',
         fileName: 'Server.zip',
         temp: `${os.tmpdir()}/OSFRServer`
     }).then(() => {
         busy = true;
-        Notification.show('information', 'Installing server');
+        Notification.show('success', 'Done');

```

Comment:
may want a different notification here as the server is not extracted at this point. Perhaps "Extracting Server Files"?

Reply:
The extraction for the server files is so fast that u dont even need a notification, otherwise u will stack notifications which makes it ugly, i like to keep it clean

Reply:
we'd still want this to happen after extraction is truly complete. Perhaps it and the "Downloading Client Files" notification can be combined into 1. 
for example "Server installation complete, downloading client files"

Reply:
Wouldn't that be to long tho?

Reply:
![Screenshot (26)](https://github.com/Open-Source-Free-Realms/OSFR-Launcher/assets/95110502/8fac7328-6e6d-4193-bf7c-251606448cc4)

Reply:
Sorry, just saw this.
if the notification is too long, could we break it up using a newline?

Reply:
I was thinking about changing the "Downloading" parts a orange 'warning' instead of the blue one, Whats your feedback on that?

Reply:
Uploaded a dev branch: https://github.com/Stimpy7314/OSFR-Launcher/tree/dev

Install it and let me know if u like the changes

Reply:
Added a directx9 detection on the dev brench, it was added by the original dev already but it wasnt working or idk? Users were still needed to download directx9 even tho it was added already.